arxiv/base/static/sass/_custom.sass

```diff
@@ -212,3 +212,21 @@ tr.is-warning
 
 .pagination-list li + li // fixes override from content li + li
   margin: 0
+
+/* flash notifications */
+
+.notifications
+  margin-top: 5px
+
+.notification
+  padding: 0.25em 0.5em 0.25em 0.5em
+  width: 70%
+  margin-left: auto
+  margin-right: auto
+  margin-bottom: 5px !important
+
+.notification .notification-message
+  font-size: 0.8em
+
+.notification .notification-title
+  font-weight: 600
```

Comment:
definitely can use `.has-text-weight-bold` or specify heading level as appropriate (better semantic HTML for a11y)